client/Wayland/wlf_input.c

```diff
@@ -152,13 +151,14 @@ BOOL wlf_handle_pointer_axis(freerdp* instance, const UwacPointerAxisEvent* ev)
 
 	input = instance->input;
 
-	direction = wl_fixed_to_int(ev->value);
+	direction = wl_fixed_to_double(ev->value) * 10.0;
```

Comment:
This is the only relevant change in the pr.
you just multiply the default value by 10.

Everything else down can be removed, as it does the same as the old code. (get an absolute value `step` and add it to the mask)

Reply:
No, the code changes are required. Please see the following comparisson:

```
axis: 0 | value: -320 | direction/new:  255 | flags/new: 0x2ff | flags/old: 0x340
axis: 0 | value: -256 | direction/new:  255 | flags/new: 0x2ff | flags/old: 0x300
axis: 0 | value: -192 | direction/new:  192 | flags/new: 0x2c0 | flags/old: 0x2c0
axis: 0 | value: -128 | direction/new:  128 | flags/new: 0x280 | flags/old: 0x280
axis: 0 | value:  -64 | direction/new:   64 | flags/new: 0x240 | flags/old: 0x240
axis: 0 | value:    0 | direction/new:    0 | flags/new: 0x200 | flags/old: 0x200
axis: 0 | value:   64 | direction/new:  -64 | flags/new: 0x3c0 | flags/old: 0x340
axis: 0 | value:  128 | direction/new: -128 | flags/new: 0x380 | flags/old: 0x380
axis: 0 | value:  192 | direction/new: -192 | flags/new: 0x340 | flags/old: 0x3c0
axis: 0 | value:  256 | direction/new: -256 | flags/new: 0x300 | flags/old: 0x300
axis: 0 | value:  320 | direction/new: -256 | flags/new: 0x300 | flags/old: 0x340
```